example/src/components/InstallTabs.tsx

```diff
@@ -20,7 +20,7 @@ export default function InstallTabs() {
     },
     {
       name: 'no build',
```

Comment:
I think you should also bump it for css part here

Reply:
done